p2p_wallet/Scenes/CreateOrRestoreWallet/RestoreWallet/EnterSeedViewController/EnterSeedInfo/EnterSeedInfoNavigationBar.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  EnterSeedInfoNavigationBar.swift
+//  p2p_wallet
+//
+//  Created by Andrew Vasiliev on 18.11.2021.
+//
+
+import UIKit
+
+final class EnterSeedInfoNavigationBar: WLNavigationBar {
```

Comment:
subview of a scene must be implemented in scene's extension
```swift
extension EnterSeedInfo {
    final class NavigationBar: WLNavigationBar
}
```
The same for EnterSeedNavigationBar